src/components/Send/index.tsx

```diff
@@ -383,8 +383,13 @@ export default function Send({ wallet }: SendProps) {
       }
 
       if (showConfirmSendModal === undefined) {
-        setShowConfirmSendModal(values)
-        return
+        if (walletInfo?.utxosByJar && values.sourceJarIndex !== undefined) {
+          values.selectedUtxos = walletInfo.utxosByJar[values.sourceJarIndex].filter((utxo) => {
+            return utxo.frozen === false
+          })
```

Comment:
Is this still needed? I guess this is a left-over from previous code changes right?

Reply:
yes indeed, it is a left-over